README.md

```diff
@@ -24,6 +24,18 @@ Building from source
 
 See the file [HACKING](HACKING.md) for more details on building from source.
 
+Quick install & forget on Ubuntu
+-----------------------
+Download Redshift using Ubuntu Software Center or launch command in terminal: `sudo apt-get install redshift-gtk`
+
+Open system menu and launch Session and Startup. Redshift should be already on the list. Tick the box next to it's name. From the next login, Redshift will adjust your screen automatically.
```

Comment:
What system menu do you mean?

Reply:
@Kasijjuf I think he uses Xfce. I don't see anything titled "Session and Startup" in the system settings of vanilla Ubuntu 16.04.